### PR TITLE
Adds new build system to repo

### PR DESCRIPTION
**Pull request by ThadHouse**

### CODE REVIEW

File: src/main/native/include/tcpsockets/TCPAcceptor.h [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
    ------------------------------------------
 
-   Copyright � 2013 [Vic Hargrave - http://vichargrave.com]
+   Copyright � 2013 [Vic Hargrave - http://vichargrave.com]
```
Comment (PeterJohnson) [inline src/main/native/include/tcpsockets/TCPAcceptor.h]: Corruption?

File: src/main/native/include/tcpsockets/TCPConnector.h [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
    ------------------------------------------
 
-   Copyright � 2013 [Vic Hargrave - http://vichargrave.com]
+   Copyright � 2013 [Vic Hargrave - http://vichargrave.com]
```
Comment (PeterJohnson) [inline src/main/native/include/tcpsockets/TCPConnector.h]: Corruption?

File: src/main/native/include/tcpsockets/TCPStream.h [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
    ------------------------------------------
 
-   Copyright � 2013 [Vic Hargrave - http://vichargrave.com]
+   Copyright � 2013 [Vic Hargrave - http://vichargrave.com]
```
Comment (PeterJohnson) [inline src/main/native/include/tcpsockets/TCPStream.h]: Corruption?

File: src/dev/native/cpp/main.cpp [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#include "llvm/StringRef.h"
+#include <iostream>
+
+int main() {
+    llvm::StringRef v1("Hello");
+    std::cout << v1.lower() << std::endl;
+}
```
Comment (PeterJohnson) [inline src/dev/native/cpp/main.cpp]: EOL at EOF

File: settings.gradle [hunk 5]
```diff
@@ -0,0 +1 @@
+include 'gmock'
```
Comment (PeterJohnson) [inline settings.gradle]: EOL at EOF

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+# Contributing to ntcore
```
Comment (ThadHouse) [inline CONTRIBUTING.md]: Change to wpiutil

File: src/main/native/include/tcpsockets/TCPAcceptor.h (hunk 1 above)
Comment (calcmogul) [inline src/main/native/include/tcpsockets/TCPAcceptor.h]: The copyright symbol got mangled. UTF problems? We could either find the right symbol or replace it with "(c)".

File: settings.gradle (hunk 5 above)
Comment (calcmogul) [reply settings.gradle]: Yeah... we should get wpiformat set up.

**Review by calcmogul — DISMISSED** [review]

**Comment by ThadHouse** [comment]
Fixed all the EOL and the copyright issues.

**Review by PeterJohnson — APPROVED** [review]

**Pull request merged**